### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by vanyle**
For now, there are 3 tests.
- A test that the window can be opened.
- A test that python code can be executed in a cell.
- A test that a block can be moved.

Do not use your mouse or keyboard during the test !

Note that the `running_python` test fails as this code base comes from the master branch where this feature does not exist yet.

### CODE REVIEW

File: .github/workflows/python-tests.yml [hunk 1]
```diff
@@ -33,4 +33,4 @@ jobs:
     - name: Test with pytest
       run: |
         pip install -r requirements-dev.txt
-        pytest tests
+        pytest tests/unit
```
Comment (MathisFederico) [inline .github/workflows/python-tests.yml:36]: Not only unit tests should be run, the passed flag should mean all tests are passed

File: tests/integration/test_blocks.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    # Click outside the block to lose focus of the previous block.
+    # This will need to be changed by the click to the run button.
+    QtTest.QTest.mouseClick(ocb_widget,Qt.MouseButton.LeftButton)
+    QApplication.processEvents()
+
+    # When the execution becomes non-blocking for the UI, a refactor will be needed here.
+    result = test_block.stdout.strip()
+    
+    check.equal(expected_result,result)
+    wnd.close()
```
Comment (MathisFederico) [inline tests/integration/test_blocks.py:76]: This should be added in kernel branch only, it fails on master

File: tests/integration/test_blocks.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+def test_open_file():
+    """
+        The application loads files properly. 
+    """
+
+    wnd = OCBWindow()
+    file_example_path = "./tests/testing_assets/example_graph1.ipyg"
+    subwnd = wnd.createNewMdiChild(os.path.abspath(file_example_path))
+    subwnd.show()
+    wnd.close()
```
Comment (MathisFederico) [inline tests/integration/test_blocks.py:169]: Missing line at the end of file

**Review by MathisFederico — CHANGES_REQUESTED** [review]
A few changes and it will be good to go !
A very nice work, this will save tons of time !

File: tests/integration/test_blocks.py (hunk 2 above)
Comment (vanyle) [reply tests/integration/test_blocks.py]: Is commenting it ok ?

Comment (MathisFederico) [reply tests/integration/test_blocks.py]: Yeah sure

**Review by MathisFederico — APPROVED** [review]
Nice ! This is an awesome add !

**Pull request merged**